SeisoLLC/easy_infra
# Add tfsec and run prior to executing terraform

# Contributor Comments
This adds `tfsec` to the image and runs (interactively and non-interactively) prior to executing `terraform` commands.

## Testing
1. Build the new image via `make build`
1. Test the container in a few ways:
    1. Ensure the updates still work via `make update`
    1. 
    1. 
    1. 
    1. 

## Pull Request Checklist
Thank you for submitting a contribution to `easy_infra`!

In order to streamline the review of your contribution we ask that you review and comply with the below requirements:
 - [X] Ensure that `make build` completes successfully
 - [X] Rebase your branch against the latest commit of the target branch, which likely should be `master`
 - [X] If there is an issue associated with your Pull Request, align your PR title with [this documentation](https://help.github.com/en/articles/closing-issues-using-keywords)
